packages/spectral/src/types/PollingTriggerDefinition.ts

```diff
@@ -0,0 +1,73 @@
+import type { AxiosRequestConfig, AxiosResponse } from "axios";
+import type {
+  ActionDisplayDefinition,
+  TriggerEventFunction,
+  Inputs,
+  TriggerResult,
+  ConfigVarResultCollection,
+  TriggerPayload,
+  ActionDefinition,
+  ActionContext,
+  ActionInputParameters,
+} from ".";
+
+export type PollingContext<TInputs extends Inputs = Inputs> = {
```

Comment:
Extending `ActionContext` here instead of using an intersection as the `context` type on line 33 will prevent inadvertent property overrides with future changes.